sagemaker-pipelines/time_series_forecasting/amazon_forecast_pipeline/sm_pipeline_with_amazon_forecast.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<p>When you say "The current Scikit-learn image", can you give a date? Ie, "As of February 2022, the current Scikit-learn image ..."</p><p>That way, if it's updated in the future, customers will know this may be an old statement.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>